Gemfile.lock

```diff
@@ -603,4 +603,4 @@ DEPENDENCIES
   webrick (~> 1.4, >= 1.4.2)
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```

Comment:
There is no need for this change.  We will need to upgrade bundler in the docker images to support this